src/metpy/io/metar.py

```diff
@@ -87,33 +87,36 @@ def parse_metar_to_dataframe(metar_text, *, year=None, month=None):
-    'longitude': Longitude of the observation, measured in degrees
-    'elevation': Elevation of the observation above sea level, measured in meters
-    'date_time': Date and time of the observation, datetime object
-    'wind_direction': Direction the wind is coming from, measured in degrees
-    'wind_spd': Wind speed, measured in knots
-    'current_wx1': Current weather (1 of 3)
-    'current_wx2': Current weather (2 of 3)
-    'current_wx3': Current weather (3 of 3)
-    'skyc1': Sky cover (ex. FEW)
-    'skylev1': Height of sky cover 1, measured in feet
-    'skyc2': Sky cover (ex. OVC)
-    'skylev2': Height of sky cover 2, measured in feet
-    'skyc3': Sky cover (ex. FEW)
-    'skylev3': Height of sky cover 3, measured in feet
-    'skyc4': Sky cover (ex. CLR)
-    'skylev4:': Height of sky cover 4, measured in feet
-    'cloudcover': Cloud coverage measured in oktas, taken from maximum of sky cover values
-    'temperature': Temperature, measured in degrees Celsius
-    'dewpoint': Dew point, measured in degrees Celsius
-    'altimeter': Altimeter value, measured in inches of mercury, float
-    'current_wx1_symbol': Current weather symbol (1 of 3), integer
-    'current_wx2_symbol': Current weather symbol (2 of 3), integer
-    'current_wx3_symbol': Current weather symbol (3 of 3), integer
-    'sea_level_pressure': Sea level pressure, derived from temperature, elevation
-    and altimeter value, float
+
+    * 'station_id': Station Identifier (ex. KLOT)
+    * 'latitude': Latitude of the observation, measured in degrees
+    * 'longitude': Longitude of the observation, measured in degrees
+    * 'elevation': Elevation of the observation above sea level, measured in meters
+    * 'date_time': Date and time of the observation, datetime object
+    * 'wind_direction': Direction the wind is coming from, measured in degrees
+    * 'wind_speed': Wind speed, measured in knots
+    * 'current_wx1': Current weather (1 of 3)
+    * 'current_wx2': Current weather (2 of 3)
+    * 'current_wx3': Current weather (3 of 3)
+    * 'low_cloud_type': Low-level sky cover (ex. FEW)
+    * 'low_cloud_level': Height of low-level sky cover, measured in feet
+    * 'medium_cloud_type': Medium-level sky cover (ex. OVC)
+    * 'medium_cloud_level': Height of medium-level sky cover, measured in feet
+    * 'high_cloud_type': High-level sky cover (ex. FEW)
+    * 'high_cloud_level': Height of high-level sky cover, measured in feet
+    * 'highest_cloud_type': Highest-level Sky cover (ex. CLR)
+    * 'highest_cloud_level:': Height of highest-level sky cover, measured in feet
+    * 'cloud_coverage': Cloud cover measured in oktas, taken from maximum of sky cover values
+    * 'air_temperature': Temperature, measured in degrees Celsius
+    * 'dew_point_temperature': Dew point, measured in degrees Celsius
+    * 'altimeter': Altimeter value, measured in inches of mercury, float
+    * 'current_wx1_symbol': Current weather symbol (1 of 3), integer
```

Comment:
How about instead of just `integer`, we go with `WMO integer code` and reference [WMO 306 Vol 1 Part A (Attachment IV)](https://library.wmo.int/index.php?lvl=notice_display&id=13617#.YMfXhi9h1hE)? Would also be good to update the our [existing reference](https://github.com/Unidata/MetPy/blob/06c3bac994911bb22ed9a7be5c3b752f3cd1d628/src/metpy/plots/wx_symbols.py#L186-L192) since it looks like they moved it on us?

Reply:
(and of course in the other places as well)